app/controllers/katello/api/v2/content_views_controller.rb

```diff
@@ -54,6 +54,7 @@ def filtered_associations
     param :composite, :bool, :desc => N_("Filter only composite content views")
     param :without, Array, :desc => N_("Do not include this array of content views")
     param :name, String, :desc => N_("Name of the content view"), :required => false
+    param :label, String, :desc => N_("Label of the content view"), :required => false
```

Comment:
seems unnecessary given you can use `search`

Reply:
I'd propose leaving it here unless you see an issue with it?

Reply:
I am ok leaving it. May be you could add this parameter to environments controller index also?

Reply:
Just to be consistent